helix-term/src/application.rs

```diff
@@ -471,7 +471,14 @@ impl Application {
                 }
             }
             signal::SIGCONT => {
-                self.claim_term().await.unwrap();
+                // Copy/Paste from same issue from neovim:
+                // https://github.com/neovim/neovim/issues/12322
```

Comment:
I 
```suggestion
                // https://github.com/neovim/neovim/pull/13084
```

we should link the PR instead of the issue here. I was confused at first because there were other patches mentioned in the issue. The PR back links to the issue so it's easy to discover too but it took a bit of digging for me to find the PR